rfc/rfc-0000-lightweight-rfc-process.md

```diff
@@ -0,0 +1,67 @@
+# RFC-0000: Lightweight RFC Process for riff
+
+**Authors:** Swapnil Bawaskar
+
+**Status:** **Active** | Accepted | Rejected | Superseded
+
+**Pull Request URL:** https://github.com/projectriff/riff/pull/1358
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** N/A
+
+
+## Problem
+As the scope of the project expands and the number of moving pieces increases, we need a way for contributors to propose designs for significant changes, and encourage discussion and feedback in a PR forum before proceeding with implementation.
+
+This document will refer to that process as the “Request For Comments (RFC) process”.
+
+## Solution
+The proposed solution to address the problem described above is to have an individual author or a group of authors submit a proposal to the community in the form of a RFC PR with a RFC markdown file, both called rfc-xxxx-<some-name>.md, in order to gather feedback and achieve consensus. The RFC follows the same format as used by this proposal.
+
+Much inspiration for this proposal has been drawn from [Apache Geode's RFC process](https://cwiki.apache.org/confluence/display/GEODE/Lightweight+RFC+Process), which in turn draws from Phil Calçado’s [Structured RFC Process](https://philcalcado.com/2018/11/19/a_structured_rfc_process.html).
+
+All RFCs are submitted via PRs to the [github.com/projectriff/riff](https://github.com/projectriff/riff) repo which will be merged when consensus has been reached about the state ("Accepted/Rejected").
+
+### Collaboration
+Comments and feedback should be provided on the PR.
+
+Authors should address all comments. This doesn't mean every comment and suggestion must be accepted and incorporated, but they should be carefully read and responded to.
+
+Every RFC is in one of the following phases:
+* **Active**: The RFC has been drafted and is open for comments. When making the PR, the doc should start out in this state.
+* **Accepted**: All the comments have been addressed and the proposed changes have been agreed upon. The implementation may start after this point.
+* **Rejected**: The changes proposed on this RFC were not agreed upon and no implementation will follow.
+* **Superseded**: The changes proposed on this RFC aren't in effect anymore, the document may be kept for historical purposes and there is a new RFC that’s more current.
```

Comment:
I noticed in some IETF language, this corresponds roughly to what they label as "Deprecated".

Reply:
If we change "Superseded" -> "Deprecated", I am assuming that 
```
**Superseded by:**
**Supersedes:**
```
are not impacted.

Reply:
I think those fields could remain as-is, but it should also be possible for an RFC to become "Deprecated" simply due to becoming obsolete without explicitly being superseded by another RFC.

Reply:
Incorporated.